### ISSUE TITLE
Module project: Debugging enhancements

### ISSUE THREAD

**Issue by trondhindenes** [adamdriscoll/poshtools#302]
When debugging a module project there doesn't seem to be any "linkage" to the actual module project - I have to manually import the module's psd1 or psm1 file manually. I think it would be a nice addition to get the following debug options:
1. Auto-load/reload the module when debugging so that the module is available in the built-in repl and any "helper scripts"
2. Set path to the current project's folder so that "helper scripts" could just reference any scripts in the projects by filename instead of full path.
In short, more granular control of the debugging experience would be great.

**Comment by adamdriscoll** [comment]
Great idea! It would make the module project much more useful.
